src/components/common/searchbar.tsx

```diff
@@ -0,0 +1,29 @@
+import { FC } from 'react';
+import SearchIcon from "../../assets/search.svg";
+import { InputText } from "primereact/inputtext";
+import { Button } from 'primereact/button';
```

Comment:
InputText should be a reusable component.
Also use the reusasble button component here.